src/PrestaShopBundle/Resources/config/services/adapter/product/_pack.yml

```diff
@@ -27,10 +30,14 @@ services:
       - name: tactician.handler
         command: PrestaShop\PrestaShop\Core\Domain\Product\Pack\Command\RemoveAllProductsFromPackCommand
 
-  prestashop.adapter.product.pack.command_handler.get_packed_products_handler:
-    class: PrestaShop\PrestaShop\Adapter\Product\Pack\QueryHandler\GetPackedProductsHandler
+  prestashop.adapter.product.pack.command_handler.get_packed_products_details_handler:
```

Comment:
```suggestion
  prestashop.adapter.product.pack.query_handler.get_packed_products_details_handler:
```

Reply:
sorry it was wrongly named initially We match the service name with the namespace